docfx_project/ziti/clients/linux.md

```diff
@@ -1,59 +1,147 @@
 ## Linux
 
-The linux tunneler is capable of operating in both transparent and proxy modes. In transparent mode the tunneler will
-interact with the operating system to establish intercept points. These intercept points correspond directly to the
-service definitions in place.
+### The Tunneler CLI
 
-`ziti-tunnel` supports several intercept modes, which are specified with a sub-command.
-Typically you will run `ziti-tunnel` with the _run_ sub-command, which determines
-the preferred intercept mode based on the kernel drivers that are available on the
-host.
+`ziti-edge-tunnel` is the general purpose tunneler CLI and can also run as a systemd daemon. For the best overall experience, please use the preferred tunneler `ziti-edge-tunnel` described here.
 
-    $ sudo ziti-tunnel run ziti.json
-    [   0.000]    INFO ziti/tunnel/intercept/tproxy.New: tproxy listening on 127.0.0.1:33641
-    [   0.006]    INFO ziti/tunnel/cmd/ziti-tunnel/subcmd.run: using tproxy interceptor
-    ...
+The purpose of the tunneler is to configure host access. This means all users and all processes on the host will share the same level of access. This is accomplished by configuring the OS to have an on-board OpenZiti DNS nameserver and IP routes for authorized OpenZiti Services.
 
-When started on a host that lacks __iptables__, the tproxy intercept mode initializer
-fails and `ziti-tunnel` attempts to use the tun intercept mode:
+### Installation and Upgrade
 
-    $ sudo ziti-tunnel run ziti.json
-    [   0.001]    INFO ziti/tunnel/intercept/tproxy.New: tproxy listening on 127.0.0.1:37313
-    [   0.001]    INFO ziti/tunnel/cmd/ziti-tunnel/subcmd.run: tproxy initialization failed: failed to initialize iptables handle: exec: "iptables": executable file not found in $PATH
-    [   0.009]    INFO ziti/tunnel/cmd/ziti-tunnel/subcmd.run: using tun interceptor
+[The latest release](https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/) of `ziti-edge-tunnel` is distributed as a binary executable from GitHub. The upgrade procedure is identical to the installation procedure.

```

Comment:
Will you file an issue to track when this is part of package managers to come back here and update this with "easier" instructions? this script indicates bash but is it `sh` compatible? Would/should we add a helper function to the ziti-cli-scripts.sh to add getting the linux binary? (this function basically it seems?)

Reply:
https://github.com/openziti/ziti-doc/issues/122

Reply:
This script is `sh` compatible and doesn't use any modern BASHisms. My preference is to see the few lines of clear, pasteable shell code over a comparatively opaque shell function.